src/modules/jsUtils.js

```diff
@@ -0,0 +1,94 @@
+/**
```

Comment:
We have a `utils` module already, that has this kind of utility functions. Maybe we can place this in there, if we keep growing that file, then we should separate it, but right now, it only has js utils and some location utilities. It is not too big.